Makefile

```diff
@@ -423,18 +423,23 @@ endif
 # 	$(if $(value WAIT),|| { echo "Terminated with $$?"; wait "$$PID"; },)
 
 kcp/bin/kcp:
-	bash ./bootstrap/install-kcp-with-plugins.sh
+	bash ./bootstrap/bootstrap-kubestellar.sh --deploy false
 
 .PHONY: e2e-test-kubestellar-syncer
 e2e-test-kubestellar-syncer: WORK_DIR ?= $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 e2e-test-kubestellar-syncer: TEST_ARGS ?= 
 e2e-test-kubestellar-syncer: KIND_CLUSTER_NAME ?= e2e-kubestellar
-e2e-test-kubestellar-syncer: e2e-test-kubestellar-syncer-cleanup kcp/bin/kcp
-	export PATH=$(PWD)/kcp/bin:$$PATH && \
+e2e-test-kubestellar-syncer: export PATH := $(PWD)/kcp/bin:$(PATH)
```

Comment:
The "export" is not required here. Variables with non-problematic names are automatically exported (https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html#index-variables_002c-exporting; I tested this on MacOS too).